src/csl/rendering/mod.rs

```diff
@@ -560,13 +566,16 @@ fn render_date_part<T: EntryLike>(
                 if val != 1
                     || !ctx
                         .style
-                        .lookup_locale(|l| {
-                            Some(
-                                l.style_options
-                                    .and_then(|o| o.limit_day_ordinals_to_day_1)
-                                    .unwrap_or_default(),
-                            )
-                        })
+                        .lookup_locale(
+                            |l| {
+                                Some(
+                                    l.style_options
+                                        .and_then(|o| o.limit_day_ordinals_to_day_1)
+                                        .unwrap_or_default(),
+                                )
+                            },
+                            None,
```

Comment:
You know the drill: I find this none suspicious, same below in the file

Reply:
Yeah, it also seemed weird for me, but I decided to keep it as it is, because I didn't find any reasons to pass language here. Again, any discussion is welcome, I agree that I'm not the best contributor for those things as I have really small experience in bibliography standarts.

Reply:
Fixed it.

Reply:
Fixed.